src/fragments/lib/common/js/import_configuration.mdx

```diff
@@ -0,0 +1 @@
+Import and load the configuration file in your app. It's recommended you add the Amplify configuration step to your app's root entry point. For example `index.js` in React or `main.ts` in Angular.
```

Comment:
```suggestion
Import and load the configuration file in your app. It's recommended you add the Amplify configuration step to your app's root entry point. For example `index.tsx` in React or `main.ts` in Angular.
```

Reply:
as discussed offline, will keep as `index.js` to cater to the most likely beginner path (CRA without typescript flags)